ui/app/templates/components/control-group.hbs

```diff
@@ -0,0 +1,97 @@
+<div class="box is-fullwidth is-bottomless is-sideless">
+  <MessageError @model={{model}} />
+  <div class="control-group-header {{if isSuccess 'is-success'}}">
+    <p>
+      <ICon @glyph={{if isSuccess "checkmark-circled" "control-lock"}} @size={{16}} />
+      <strong data-test-banner-prefix>{{bannerPrefix}}</strong>
+      {{bannerText}}
+    </p>
+  </div>
+  {{#if (and currentUserIsRequesting model.approved)}}
+    <ControlGroupSuccess @model={{model}} @controlGroupResponse={{controlGroupResponse}} />
+  {{else}}
+    <div class="control-group">
+      {{#if model.requestEntity.canRead}}
+        <a href={{href-to 'vault.cluster.access.identity.show' 'entities' model.requestEntity.id 'details'}}>
+          {{requestorName}}
+        </a>
+      {{else}}
+        {{requestorName}}
+      {{/if}}
+      {{#if currentUserIsRequesting}}are{{else}}is{{/if}}
+      {{#if model.approved}}
+        authorized to access <code>{{model.requestPath}}</code>
+      {{else}}
+        requesting access to <code>{{model.requestPath}}</code>
+      {{/if}}
+
+      {{#if currentUserIsRequesting}}
+        <div class="message is-list is-highlight has-copy-button" tabindex="-1">
+          <HoverCopyButton @alwaysShow=true @copyValue={{model.id}} />
+          <div class="message-body">
+            <h4 class="title is-7 is-marginless">
+              Accessor
+            </h4>
+            <code class="is-word-break">{{model.id}}</code>
+          </div>
+        </div>
+      {{/if}}
+
+      <div class="authorizations">
+        {{#if (gt model.authorizations.length 0)}}
+        <span class="has-text-success">
+          <ICon @glyph="checkmark-circled-outline" @size={{16}} />
```

Comment:
This looks really weird now (`ICon`), at what point can you move this to just `Icon`?

Reply:
Good eye! [At least according to the RFC](https://github.com/emberjs/rfcs/blob/master/text/0311-angle-bracket-invocation.md#tag-name), I think we can now use `<Icon>`:

> Another benefit of the capital letter rule is that we can now support component names with a single word, such as <Button>, <Modal> and <Tab>.

Does that work in practice?

Reply:
Sadly not until the upgrade to 3.2 - single word components is the only omission from the polyfill - https://github.com/rwjblue/ember-angle-bracket-invocation-polyfill#limitations